packages/zora/src/Zora.ts

```diff
@@ -105,8 +107,14 @@ export const mint = async (
       ] as Address[])
     : contractAddress
 
-  const andArray721 = []
-  const andArray1155 = []
+  const initialArray = [
+    {
+      quantity: formatAmount(amount),
+    },
+  ]
+  const andArray721: AndArrayItem[] = [...initialArray]
+  const andArray1155: AndArrayItem[] = [...initialArray]
```

Comment:
[nit] Can convert the `initialArray` into the object so you don't need to spread the array

```
const quantityCheck = {
  quantity: formatAmount(amount),
}
const andArray721: AndArrayItem[] = [quantityCheck]
const andArray1155: AndArrayItem[] = [quantityCheck]

Reply:
updated. good catch.